spec/features/edit_course_details_spec.rb

```diff
@@ -17,27 +18,38 @@ def when_i_visit_the_course_details_page
   end
 
   def and_i_enter_valid_parameters
-    page_object = PageObjects::Trainees::CourseDetails.new
-    page_object.course_title.set "Test Course"
-    page_object.course_phase.set "11-16"
-    set_date_fields(page_object, "programme_start_date", "10/01/2021")
-    page_object.programme_length.set "1 year"
-    set_date_fields(page_object, "programme_end_date", "10/01/2022")
-    page_object.allocation_subject.set "Maths"
-    page_object.itt_subject.set "Maths"
-    page_object.employing_school.set "Hillside College"
-    page_object.placement_school.set "Northwich Academy"
-    page_object.submit_button.click
+    course_details_page.course_title.set "Test Course"
+    course_details_page.course_phase.set "11-16"
+    set_date_fields("programme_start_date", "10/01/2021")
+    course_details_page.programme_length.set "1 year"
+    set_date_fields("programme_end_date", "10/01/2022")
+    course_details_page.allocation_subject.set "Maths"
+    course_details_page.itt_subject.set "Maths"
+    course_details_page.employing_school.set "Hillside College"
+    course_details_page.placement_school.set "Northwich Academy"
+    course_details_page.submit_button.click
   end
 
   def then_i_am_redirected_to_the_summary_page
     expect(page.current_path).to eq("/trainees/#{@trainee.id}")
   end
 
-  def set_date_fields(page_object, field_prefix, date_string)
+  def set_date_fields(field_prefix, date_string)
     day, month, year = date_string.split("/")
-    page_object.send("#{field_prefix}_day").set day
-    page_object.send("#{field_prefix}_month").set month
-    page_object.send("#{field_prefix}_year").set year
+    course_details_page.send("#{field_prefix}_day").set day
+    course_details_page.send("#{field_prefix}_month").set month
+    course_details_page.send("#{field_prefix}_year").set year
+  end
+
+  def and_the_updated_values_are_visible
```

Comment:
```suggestion
  def and_the_updated_course_title_are_visible
```

Reply:
My intention here is that we'd check the other values in this too but I didn't add them as yet because they're going to change so much.